admin/src/identity-provider/identity-provider.service.ts

```diff
@@ -119,16 +124,16 @@ export class IdentityProviderService {
     Object.assign(existingProvider, providerToSave);
 
     // Save the updated provider
-    const identityProviderResponse =
+    const identityProviderFromDb =
       await this.identityProviderRepository.save(existingProvider);
 
-    return identityProviderResponse;
+    const hasGristPublicationSucceeded =
+      await this.publishIdentityProvidersToGrist();
+
+    return { identityProviderFromDb, hasGristPublicationSucceeded };
```

Comment:
nitpick: it could be more elegant to return the same payload for the create and the update functions:
```suggestion
    return { identityProviderId, hasGristPublicationSucceeded };
```

Plus "frowDb" describe an internal usage that should not be exposed outside of this function.